pkg/controller/elasticsearch/client/model.go

```diff
@@ -66,16 +68,25 @@ func (n Nodes) Names() []string {
 
 // Node partially models an Elasticsearch node retrieved from /_nodes
 type Node struct {
-	Name  string   `json:"name"`
-	Roles []string `json:"roles"`
-	JVM   struct {
+	Name    string   `json:"name"`
+	Version string   `json:"version"`
+	Roles   []string `json:"roles"`
+	JVM     struct {
 		StartTimeInMillis int64 `json:"start_time_in_millis"`
 		Mem               struct {
 			HeapMaxInBytes int `json:"heap_max_in_bytes"`
 		} `json:"mem"`
 	} `json:"jvm"`
 }
 
+func (n Node) isV7OrAbove() (bool, error) {
+	v, err := version.Parse(n.Version)
+	if err != nil {
+		return false, errors.Wrap(err, fmt.Sprintf("unable to parse node version %s", n.Version))
+	}
+	return v.Major > 7, nil
```

Comment:
I think there's a bug here, `isV7OrAbove` actually returns `false` if the major version of the cluster is 7.

Reply:
Wow, indeed !

Reply:
Thanks for fixing it!